lib/chippy_web/templates/layout/app.html.eex

```diff
@@ -12,7 +12,7 @@
       <section class="container">
         <nav role="navigation">
           <ul>
-            <li><a href="/profile" id="user_id"><%= Plug.Conn.get_session(@conn, :user_id) || "Edit profile" %></a></li>
+            <li><a href="<%= Routes.page_path(ChippyWeb.Endpoint, :profile) %>?next=<%= @conn.request_path %>" id="user_id"><%= Plug.Conn.get_session(@conn, :user_id) || "Edit profile" %></a></li>
```

Comment:
I was just reading about `link` helpers: https://hexdocs.pm/phoenix_html/Phoenix.HTML.Link.html#link/2

Not sure if it provides much benefit here though...

Reply:
Good idea! I'll give it a try...